content/docs/guides/contribute/conventions/front.en.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Front-end"
+linkTitle: "Front-end"
+description: "Coding style guide and best practices for front-end"
```

Comment:
This is supposed to be the english page but the entire text is in french, maybe a bad copy-paste ?

Reply:
Nope, as this section is currently on rewrite/conception (along actual front refacto), we write it in french, and translation will come after stabilization !

Reply:
Resolve is at your discretion :heart:

Reply:
Hmm seems weird to me but ok 🤔